{"name": "test_2_TypeAddClass", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <testcases.test_AddClassification.TestAddClassification object at 0x053FECB8>\ntest_logsystem = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"1c96fe59-963f-4400-a927-65ad0e595dff\")>\n\n    def test_2_TypeAddClass(self,test_logsystem):\n        self.page1 = TestLogPage(test_logsystem)\n        self.page1.type_new_sort_name(\"smart\")\n        self.page1.click_save_sort_name()\n        try:\n            self.page1.click_logclass()\n            time.sleep(2)\n>           assert \"smart\" in self.page1.check_logclass()\nE           TypeError: 'tuple' object is not callable\n\ntest_AddClassification.py:36: TypeError"}, "start": 1628999885511, "stop": 1628999887796, "uuid": "a8bb4b9d-3c80-444a-a8b3-2337ced5fde3", "historyId": "d242da973f70d4de115e15a504d2bf96", "testCaseId": "34cc12a412b21ac606dd332c398f98e7", "fullName": "testcases.test_AddClassification.TestAddClassification#test_2_TypeAddClass", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_AddClassification"}, {"name": "subSuite", "value": "TestAddClassification"}, {"name": "host", "value": "DESKTOP-S2QKEKE"}, {"name": "thread", "value": "9612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_AddClassification"}]}